#request
int32 address
---
#response
int32 value
bool success
